src/pages/LibrariesPage.tsx

```diff
@@ -69,17 +71,22 @@ export default function LibrariesPage() {
         return list;
     }
 
-function sortLibraries() {
-    const sortedLibraries = [...filteredLibraries].sort((a, b) => {
-        if (sortAscending) {
-            setSortAscending(false);
-            return a.name.toLowerCase() > b.name.toLowerCase() ? 1 : -1
-        }
-        setSortAscending(true);
-        return a.name.toLowerCase() < b.name.toLowerCase() ? 1 : -1
-    })
-    setFilteredLibraries(sortedLibraries);
-}
+    function sortLibraries() {
+        const sortedLibraries = [...filteredLibraries].sort((a, b) => {
+            if (sortAscending) {
+                setSortAscending(false);
+                return a.name.toLowerCase() > b.name.toLowerCase() ? 1 : -1
+            }
+            setSortAscending(true);
+            return a.name.toLowerCase() < b.name.toLowerCase() ? 1 : -1
+        })
+        setFilteredLibraries(sortedLibraries);
+    }
+
+    const onDeleteLibrary = async (id: string) => {
+        await libraryApi.deleteLibrary(id);
```

Comment:
We'll definitely want a warning/please confirm message in the final product (if not doing this now we should probably have a ticket), but at the very least right now, having an alert for "X Library deleted" would be helpful.

Reply:
Yup agreed - https://gaming-backlog.atlassian.net/browse/GB-117! Added the alert for now


https://github.com/ksadoff/gaming-backlog-frontend/assets/24842433/451bcace-5e53-40e6-a0a3-e104ed9b8b7d